fieldImages/src/main/java/edu/wpi/fields/FieldImages.java

```diff
@@ -4,22 +4,23 @@
 
 package edu.wpi.fields;
 
+@Deprecated(since = "2023")
```

Comment:
We can probably just delete this if we’re replacing it in this PR.  I know of no usage of this class at present.